docs/supported_lockfiles.md

```diff
@@ -0,0 +1,35 @@
+# Supported Lockfiles
+
+The Phylum CLI supports processing many different lockfiles:
+
+| Lockfile type | Filenames                                                              |
+| ------------- | ---------------------------------------------------------------------- |
+| `npm`         | `package-lock.json` <br /> `npm-shrinkwrap.json`                       |
+| `yarn`        | `yarn.lock` (Version 1 + 2)                                            |
+| `pnpm`        | `pnpm-lock.yaml`                                                       |
+| `pip`         | `requirements*.txt`                                                    |
+| `pipenv`      | `Pipfile.lock`                                                         |
+| `poetry`      | `poetry.lock` (Version 1 + 2)                                          |
+| `gem`         | `Gemfile.lock`                                                         |
+| `msbuild`     | `*.csproj`                                                             |
+| `nugetlock`   | `packages.lock.json`                                                   |
+| `mvn`         | `effective-pom.xml`                                                    |
+| `gradle`      | `gradle.lockfile`                                                      |
+| `go`          | `go.sum`                                                               |
+| `cargo`       | `Cargo.lock`                                                           |
+| `spdx`        | `*.spdx.json` <br /> `*.spdx.yaml` <br /> `*.spdx.yml` <br /> `*.spdx` |
+
+:::note
+
+The lockfile type will be automatically detected based on the filename.
+
+If needed, this can be overridden with the `--lockfile-type` (`-t`) option.
+
+:::
+
+:::tip Manifest Support
+
+Lockfiles can also automatically be generated for certain manifest files.
+See [lockfile_generation](./lockfile_generation.md) for details.
+
+:::
```

Comment:
> Are we just abandoning Markdown at this point?

Yes? At least, for the Docusaurus rendered pages since they have some additional, non-standard, [Markdown Features](https://docusaurus.io/docs/markdown-features).

Reply:
> additional, non-standard, [Markdown Features](https://docusaurus.io/docs/markdown-features).

Which require training for new developers and vendor-lock us into the platform.